### ISSUE TITLE
github actions macOS-12 Xcode 14.0.1 cannot find watchsimulator

### ISSUE THREAD

**Issue by banjun** [banjun/ikemen#14]
https://github.com/banjun/ikemen/actions/runs/3503843096/jobs/5869154934

```
-> Pod installation complete! There is 1 dependency from the Podfile and 1 total pod installed.
759
  Building with `xcodebuild`.
760
    $ /usr/bin/xcodebuild clean build -workspace /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/CocoaPods-Lint-20221119-2438-gexj3n-ikemen/App.xcworkspace -scheme App -configuration Release CODE_SIGN_IDENTITY=- -sdk watchsimulator -destination id=3DF6AFAB-CD62-4560-9C31-FC3974A42C80
761
    2022-11-19 14:15:00.919 xcodebuild[3317:15278] Writing error result bundle to /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/ResultBundle_2022-19-11_14-15-0000.xcresult
762
    xcodebuild: error: Unable to find a destination matching the provided destination specifier:
763
    		{ id:3DF6AFAB-CD62-4560-9C31-FC3974A42C80 }
764
    
765
    	The requested device could not be found because no available devices matched the request.
766
    
767
    	Available destinations for the "App" scheme:
768
    		{ platform:watchOS, id:dvtdevice-DVTiOSDevicePlaceholder-watchos:placeholder, name:Any watchOS Device }
769
    		{ platform:watchOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-watchsimulator:placeholder, name:Any watchOS Simulator Device }
770
    		{ platform:watchOS Simulator, id:3DF6AFAB-CD62-4560-9C31-FC3974A42C80, OS:9.0, name:Apple Watch SE (40mm) (2nd generation) }
771
    		{ platform:watchOS Simulator, id:3C09F1C5-0AB2-4E92-A08F-9DD4AAFE8133, OS:9.1, name:Apple Watch SE (44mm) (2nd generation) }
772
```

**Comment by banjun** [comment]
41a3c14d2e62b1d45340654b9fe2bd67e3fbb647 verbose
96a63536319f1813d4a4b35b29b08ea53f91a31b revert

**Comment by banjun** [comment]
adding a simulator pair may solve this issue (it reproduced on local mac)

**Comment by banjun** [comment]
fixed in #17
